cmake/DaemonFlags.cmake

```diff
@@ -231,6 +231,16 @@ else()
 		endif()
 	endif()
 
+	if (NACL AND USE_NACL_SAIGO AND SAIGO_ARCH STREQUAL "arm")
+		# This should be set for every build type because build type flags
```

Comment:
Please add a link here to the relevant issue

Reply:
Done.